grpc-ballerina/tests/22_retry_client.bal

```diff
@@ -36,17 +32,21 @@ RetryConfiguration failingRetryConfig = {
     backoffFactor: 2,
     errorTypes: [UnavailableError, InternalError]
 };
-ClientConfiguration failingClientConfig = {
-    timeout: 1,
-    retryConfiguration: failingRetryConfig
-};
 
 final RetryServiceClient retryClient = check new("http://localhost:9112", timeout = 1, retryConfiguration = retryConfig);
 final RetryServiceClient failingRetryClient = check new("http://localhost:9112", timeout = 1, retryConfiguration = failingRetryConfig);
 
 @test:Config {enable:true}
 function testRetry() {
-    var result = retryClient->getResult("RetryClient");
+    var result = retryClient->getResult("UnavailableError");
+    if (result is Error) {
+        io:println(result);
+        test:assertFail(result.toString());
+    } else {
+        test:assertEquals(result, "Total Attempts: 4");
+    }
+
+    result = retryClient->getResult("InternalError");
     if (result is Error) {
         io:println(result);
```

Comment:
Do we need this print

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/219/commits/2a44acac6a26d011708bf186f99566cc19fa06e4